certbot/docs/using.rst

```diff
@@ -802,64 +741,116 @@ configuration file we advise you to make a backup of the file beforehand and tes
 
 .. warning:: Manually modifying files under ``/etc/letsencrypt/renewal/`` can damage them if done improperly and we do not recommend doing so.
 
+.. _references:
 
-Automated Renewals
+Reference:
```

Comment:
Should the Reference section be stashed under "Renewing Certificates", instead of under "Managing Certificates"?

Reply:
I put it under "Managing Certificates" for now since I might add to the Reference section from other sub categories in Managing Certificates in the future.